nginx-php-moodle/bin/configure-moodle.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e;
+
+# Source shtdlib
+. /usr/local/bin/shtdlib.sh
+
+echo "Waiting for PHP-FPM to come up..."
+if wait_for_success "/etc/init.d/php7.2-fpm status"; then
+    echo "PHP-FPM started!"
+
+    echo "Waiting for PostgreSQL to come up..."
+    if wait_for_success "/usr/bin/php${PHP_VERSION} -f /usr/local/bin/check_db.php"; then
+        echo "PostgreSQL started!"
+
+        # Try to upgrade if config.php found
+        if [[ -s /opt/moodle/moodle-${MOODLE_VERSION}/config.php ]]; then
```

Comment:
Would this only run the update if the MOODLE_VERSION is modified?

Reply:
No, it runs if the file doesn't exist or is empty. The environment variable is to make it compatible in the future.

Reply:
This looks like if the config.php file is found it will run an upgrade, but that means it will run the upgrade on every startup. So if the moodle version is the same does it still run the upgrade?

Reply:
Correct, the script is smart enough to do nothing if there is no code/plugin update

Reply:
Cool sounds good